core/src/table/table.tsx

```diff
@@ -101,6 +106,7 @@ export const Table = <R,>(props: TableProps<R>) => {
 			className={[
 				s.container,
 				props.fixed ? fixed.container : "",
+				props.fill ? s['container-fill']: "",
```

Comment:
in CSS Modules, you should name your class as "containerFill" and use it as s.containerFill